File: docs/manual/scala/concepts/code/docs/home/scaladsl/immutable/MutableUser.scala
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package docs.home.scaladsl.immutable

//#mutable
class MutableUser {
  private var name: String  = null
  private var email: String = null

  def getName: String = name

  def setName(name: String): Unit =
    this.name = name

  def getEmail: String = email

  def setEmail(email: String): Unit =
    this.email = email
}
//#mutable

File: docs/manual/java/concepts/code/docs/home/immutable/MutableUser.java
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package docs.home.immutable;

// #mutable
public class MutableUser {
  private String name;
  private String email;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }
}
// #mutable
